docs/challenges.rst

```diff
@@ -1,159 +1,86 @@
-
-
-Common problems with passing different challenges
-
-HTTP-01 challenge:
-* (With webroot plugin) You aren't running Certbot on your web server
-
-  Most people should install and run Certbot on their web server hosting their website, not on their laptops or some other computer.  While you can use Certbot in manual mode on a laptop and then separately set up the appropriate files on your webserver, it's not likely to be the most convenient way to get a certificate for most users.
-
-* A domain name you're requesting a certificate for isn't correctly pointed at that web server
-
-  In most cases, every name you're requesting a certificate for should already exist and be pointed to the public IP address of the server where you're requesting that certificate.  (Some alternatives exist for complex network configurations, but they're the exception rather than the rule.)
-
-* A firewall is blocking access to port 80
-
-  The certificate authority needs to be able to connect to port 80 of your server in order to confirm that you satisfied the HTTP-01 challenge.  So that needs to be publicly reachable from the Internet, and not blocked by a router or firewall.
-
-* (With webroot plugin) You specified the webroot directory incorrectly
-
-  If you used `--webroot`, you need to tell Certbot where it can put
-  files in order to have them served by your existing web server.
-  If you said your webroot for example.com was /var/www/example.com,
-  then a file placed in /var/www/example.com/.well-known/acme-challenge/testfile should appear on
-  your web site at `http://example.com/.well-known/acme-challenge/testfile` (which you can test using a web browser). (A redirection to HTTPS
+To receive a certificate from Let's Encrypt certificate authority (CA), you must pass a *challenge* to 
+prove you control each of the domain names that will be listed in the certificate. A challenge is  one of 
+three tasks that only someone who controls the domain should be able to accomplish:
+
+* Posting a specified file in a specified location on a web site (the HTTP-01 challenge)
+* Offering a specified temporary certificate on a web site (the TLS-SNI-01 challenge) 
+* Posting a specified DNS record in the domain name system (the DNS-01 challenge)
+
+It’s possible to complete each type of challenge *automatically* (Certbot directly makes the necessary 
+changes itself, or runs another program that does so), or *manually* (Certbot tells you to make a 
+certain change, and you edit a configuration file of some kind in order to accomplish it). Certbot's 
+design favors performing challenges automatically, and this is the normal case for most users of Certbot.
+
+Some plugins offer an *authenticator*, meaning that they can satisfy challenges:
```

Comment:
I'm going to punt on this one temporarily in order to hit as much of the other LHF as possible and get a functioning version of challenges.rst into your repo. 

However, I agree that there's some overlap and these sections should be restructured. The genesis of the redundancy is that challenges.rst was developed separately. Once we get the content accurate, we can look at better integration.